pkg/lockfile/lockfile_windows.go

```diff
@@ -22,48 +69,190 @@ import (
 // - There may or MAY NOT be an actual object on the filesystem created for the specified path.
 // - Even if ro, the lock MAY be exclusive.
 func createLockerForPath(path string, ro bool) (Locker, error) {
-	return &lockfile{locked: false}, nil
+	// Check if we can open the lock.
+	fd, err := openLock(path, ro)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error opening %q", path)
```

Comment:
I have a feeling this is a stutter, IE The path would show up in the err from the openLock call above.